execution-order/execution-order.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html id="stylus">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title i18n-text-append="optionsHeading">Stylus </title>
+  <link rel="stylesheet" href="/global.css">
+  <link rel="stylesheet" href="execution-order.css">
```

Comment:
Should be loaded using `require` in onclick for the button as well as draggable-list.iife.min.js and execution-order.js

Reply:
Oops, sorry, disregard this.